contracts/container/contract.go

```diff
@@ -1369,6 +1369,21 @@ func isStorageNode(ctx storage.Context, epoch int, key interop.PublicKey) bool {
 	return contract.Call(netmapContractAddr, "isStorageNode", contract.ReadOnly, key, epoch-1).(bool)
 }
 
+func nodeFromContainer(ctx storage.Context, cID interop.Hash256, key interop.PublicKey) bool {
```

Comment:
Do we have this data for non-meta containers?

Reply:
sure, every container list is registered every epoch by the alphabet, does not matter meta state